src/Features/CSharp/Portable/EditAndContinue/CSharpEditAndContinueAnalyzer.cs

```diff
@@ -197,18 +197,18 @@ internal override IEnumerable<SyntaxToken> TryGetActiveTokens(SyntaxNode node)
                 return declarator.DescendantTokens();
             }
 
+            var bodyTokens = SyntaxUtilities.TryGetMethodDeclarationBody(node)?.DescendantTokens();
+
             if (node.IsKind(SyntaxKind.ConstructorDeclaration))
             {
                 var ctor = (ConstructorDeclarationSyntax)node;
                 if (ctor.Initializer != null)
                 {
-                    return ctor.Initializer.DescendantTokens().Concat(ctor.Body.DescendantTokens());
+                    bodyTokens = ctor.Initializer.DescendantTokens().Concat(bodyTokens);
```

Comment:
I don't follow. In what cases we calculate it twice? Not that in presence of a ctor initializer we want to concatenate the tokens of the initializer with the tokens of the body, otherwise the result is only the list of body tokens.